src/management/TicketsManager.js

```diff
@@ -98,6 +98,14 @@ TicketsManager.prototype.changePassword = function(data, cb) {
  *   }
  * });
  *
+ * @param   {Object}  data
+ * @param   {String}  [data.result_url] URL the user will be redirected to once ticket is used
+ * @param   {String}  data.user_id user_id for whom the ticket should be created
+ * @param   {Integer} [data.ttl_sec] Number of seconds for which the ticket is valid before expiration
+ * @param   {Boolean} [data.includeEmailInRedirect] Whether to include the email address as part of the returnUrl in the reset_email (true), or not (false)
```

Comment:
I think we can remove `reset_email` from description. I am aware that it is same as what we have in public docs, but we can correct it here. It is also applicable to `verify_email` usecase.

How about - `Whether to include the email address as part of the result_url (true), or not (false)`